ivy/functional/ivy/creation.py

```diff
@@ -669,5 +713,10 @@ def logspace(
     ret
         Tensor of evenly-spaced values.
 
+    Examples
+    --------
+    >>> y = ivy.logspace(2,3,num=4)
+    >>> print(y)
+    ivy.array([ 100.        ,  215.443469  ,  464.15888336, 1000.        ])
```

Comment:
same thing as `eye`, not enough examples. Kindly refer to the contribution guidelines.